app/Models/Foundation/Summit/Registration/SummitRegistrationInvitation.php

```diff
@@ -139,7 +160,7 @@ public function __construct()
         $this->tags = new ArrayCollection();
         $this->member = null;
         $this->acceptance_criteria = self::AcceptanceCriteria_AllTicketTypes;
-        $this->accepted_date = null;
+        $this->status = self::Status_Pending;
```

Comment:
we need to initialize action_date in null

Reply:
done